README.md

```diff
@@ -212,7 +217,7 @@ The benefits of this approach are:
 * Allows off-chain negotiations: advertisers can bid for impressions in real time;
 * All data other than payments information is kept off-chain.
 
-Each campaign has a duration, normally in the range of 2-12 weeks. An OUTPACE channel should have 2-3 times as long of a duration, in order to allow extra time for publishers to withdraw their revenue.
+Each campaign has a duration, normally in the range of 2-12 weeks. An OUTPACE channel should have 2-3 times as long of a duration, in order to allow extra time for direct publishers to withdraw their revenue.
```

Comment:
OUTPACE channels have infinite duration in V5, so you can cahnge this to "An OUTPACE channel has indefinite lifetime, so publishers can withdraw at any point"